src/NServiceBus.Persistence.NonDurable.AcceptanceTests/NonDurableTestSetup.cs

```diff
@@ -0,0 +1,12 @@
+﻿using NServiceBus;
+using NUnit.Framework;
+
+[SetUpFixture]
+public class NonDurableTestSetup
+{
+    [OneTimeSetUp]
+    public void SetupFixture()
+    {
+        typeof(NonDurablePersistence).ToString();
```

Comment:
as this is a workaround and this code looks irrelevant without a context, should we put a note down here as well?

Reply:
I added a comment explaining this that links to https://github.com/Particular/NServiceBus/pull/6291

I'll link back the other way to this file from the PR